readyapi4j-core/src/test/java/com/smartbear/readyapi4j/RestRequestStepRecipeTest.java

```diff
@@ -394,6 +394,7 @@ public void buildRestRequestTestRecipeWithPropertyExtractor() {
         final String[] extractedProperty = {""};
         TestRecipe recipe = newTestRecipe(
                     POST(URI)
+                            .named("RestRequest")
                         .withExtractors(
                                 propertyExtractor("Endpoint", property -> extractedProperty[0] = property)))
```

Comment:
Looking at the javadoc for these methods, 'fromProperty' and 'fromResponse' sounds better to me. I can change if no objections.